README.md

```diff
@@ -0,0 +1,25 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Abdulrahman's ؛ortfolio
```

Comment:
- [ ] [OPTIONAL] Please fix the spelling error. You can also make this line a header to make it more visually appealing. Eg;
`# Abdulrahman's Portfolio`